public/count.js

```diff
@@ -245,7 +245,21 @@
 
 	if (!goatcounter.no_onload)
 		on_load(function() {
-			goatcounter.count()
+			// 1. Page is visible, count request.
+			// 2. Page is not yet visible; wait until it switches to 'visible' and count.
+			// See #487
+			if (!('visibilityState' in document) || document.visibilityState === 'visible')
+				goatcounter.count()
+			else {
+				var f = function(e) {
+					if (document.visibilityState !== 'visible')
+						return
+					document.removeEventListener('visibilitychange', f)
+					count()
```

Comment:
Should this perhaps be `goatcounter.count()`?

Reply:
Oops, thanks; I forgot to push that last fix 😅